app/models/broadcast.rb

```diff
@@ -16,6 +17,16 @@ def get_inner_body(content)
     Nokogiri::HTML(content).at("body").inner_html
   end
 
+  def banner_class
```

Comment:
can we move this in a `BroadcastHelper`? Seems a better place instead of the model object

Reply:
@rhymes Hm, so the only reason I didn't do that was because I needed access to it in the `async_info_controller` so that it can be [serialized and used](https://github.com/thepracticaldev/dev.to/pull/8361/files#diff-02a7d87efea39bec38ecc2737b8eba6cR49) in the frontend. I was under the impression that the helper can only be used in views? ...But maybe I am wrong, and it actually would be accessible from that controller?

Reply:
yeah, you can access helpers from controllers with https://api.rubyonrails.org/classes/ActionController/Helpers.html#method-i-helpers

Reply:
Addressed in last commit!